### ISSUE TITLE
Responsive data tables

### ISSUE THREAD

**Issue by thevangelist** [react-bootstrap-table/react-bootstrap-table2#79]
Feature idea.

In order to make the responsiveness of table better, we could add a flag that enables the table to be shown in key-value mode. See examples: https://css-tricks.com/responsive-data-tables/

This can be of course achieved with CSS, but it would be a better idea to provide a solution for it in the first place.

**Comment by AllenFang** [comment]
that's a cool idea. BTW, `table-responsive` also can be an alternative.
https://www.w3schools.com/bootstrap/bootstrap_ref_css_tables.asp

**Comment by prajapati-parth** [comment]
It would be great if displaying as key-value would be optional as all use cases might not require it.

**Comment by thevangelist** [comment]
Indeed, I also consider we could invent even new paradigms of display-modes. (not easy though :))

**Comment by AllenFang** [comment]
We will consider seriously to implement this feature, thanks :)
